app/views/users/sessions/new.html.erb

```diff
@@ -18,7 +18,7 @@
             <div class="space-y-6">
               <div>
                 <%= form.label :email, 'Email address' %>
-                <%= form.email_field :email, required: false, class: 'dark-form-input', data: { test_id: 'email-field', attr: 'form-validation.email', action: 'blur->form-validation#validate' } %>
+                <%= form.email_field :email, required: false, class: 'dark-form-input', autofocus: true, data: { test_id: 'email-field', attr: 'form-validation.email', action: 'blur->form-validation#validate' } %>
```

Comment:
Hey @j-mcQueen, our client side validations are causing the test failures. We're being too aggressive triggering the email field validation on blur, we can change this to only trigger on input to get around it for now.

```suggestion
                <%= form.email_field :email, required: false, class: 'dark-form-input', autofocus: true, data: { test_id: 'email-field', attr: 'form-validation.email', action: 'input->form-validation#validate' } %>
```